mygame/app/scenes/debug_screen.rb

```diff
@@ -25,6 +25,19 @@ def render(screen, state)
         }
         screen.primitives << letters
       end
+
+      # chunk = state.tick_count.idiv(30) % 4
```

Comment:
I was thinking about adding maybe several "tabs" to the debug screen.... so maybe we can add this in later again

Reply:
hmmm... I think we won't need this at all ;) I'll delete it.